kolibri/plugins/learn/assets/src/views/LearnIndex.vue

```diff
@@ -53,7 +52,9 @@
   import LearnTopNav from './LearnTopNav';
   import { ASSESSMENT_FOOTER, QUIZ_FOOTER } from './footers.js';
 
-  const RecommendedSubpageStrings = crossComponentTranslator(RecommendedSubpage);
+  // Bottom toolbar is 111px high on mobile, 113px normally.
+  // We reserve the smaller number so there is no gap on either screen size.
+  const BOTTOM_SPACED_RESERVED = 111;
```

Comment:
was this added by accident?

Reply:
Not sure exactly how that worked out as something that I added - perhaps I kept it by mistake while rebasing or something.